### PR TITLE
About Page 5.5

### PR DESCRIPTION
**Pull request by ryelle**
about

Trac ticket: https://core.trac.wordpress.org/ticket/50416

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by ryelle** [comment]
About page
about

Credits page
credits

Freedoms page
freedoms

**Comment by ryelle** [comment]
Refreshed the PR with [the latest content](https://docs.google.com/document/d/12Ag6CHVRIsX0DuGeQD9eniZCLJSNf_7rJamZfmxgZoI/edit#), and worked in design elements from the figma file to create a layout. All text combos pass AA contrast. Notable I swapped the "5.5 WordPress" to "WordPress 5.5", since we've gotten accessibility feedback on that in the past.

Here's a screenshot. The light blue sections are optional placeholders for videos, images, abstract design elements, etc. Or they can be removed, and this can go back to full-width.

Screenshot_2020-07-25 About ‹ trunk wordpress test — WordPress

The credits page will need a new design element, too. Also in this screenshot is the hover state for the nav, "Freedoms" is current hovered. (focus hasn't changed, still the same light blue outline).

Screen Shot 2020-07-25 at 14 18 04-fullpage

File: src/wp-admin/about.php [hunk 1]
```diff
@@ -40,166 +38,202 @@
...
-				<h2><?php _e( 'Your fundamental right: privacy' ); ?></h2>
-				<p><?php _e( '5.4 helps with a variety of privacy issues around the world. So when users and stakeholders ask about regulatory compliance, or how your team handles user data, the answers should be a lot easier to get right.' ); ?></p>
-				<p><?php _e( 'Take a look:' ); ?></p>
-				<ul>
-					<li><?php _e( 'Now personal data exports include users session information and users location data from the community events widget. Plus, a table of contents!' ); ?></li>
-					<li><?php _e( 'See progress as you process export and erasure requests through the privacy tools.' ); ?></li>
-					<li><?php _e( 'Plus, little enhancements throughout give the privacy tools a little cleaner look. Your eyes will thank you!' ); ?></li>
-				</ul>
+			<div class="column">
+				<h3><?php _e('The New Block Directory' ); ?></h3>
```
Comment (Zealth57) [inline src/wp-admin/about.php:124]: Should "new" be capitalized here? All other headers we're using sentence case, and while Block Directory could be capitalized as a thing, I think the "new" is just a qualifier and shouldn't be.

File: src/wp-admin/about.php [hunk 2]
```diff
@@ -40,166 +38,202 @@
...
+				<p><?php _e( 'Now it’s easier than ever to find the block you need. The new block directory is built right into the block editor, so you can install new block types to your site without ever leaving the editor.' ); ?></p>
+
+				<h3><?php _e('And so much more.' ); ?></h3>
+				<p><?php _e( 'The highlights above are a tiny fraction of the new block-editor features you’ve just installed. Open the block editor and enjoy!' ); ?></p>
```
[the remark this answers is not in the document]
Comment (marybaum) [reply src/wp-admin/about.php:128]:
```How about
				<p><?php _e( 'These highlights are a tiny fraction of the new block-editor features you’ve just installed. Open the block editor and enjoy!' ); ?></p>
```

File: src/wp-admin/about.php [hunk 3]
```diff
@@ -40,166 +38,202 @@
...
-				</div>
+			<div class="column">
+				<h3><?php _e('Inline image editing' ); ?></h3>
+				<p><?php _e( 'Crop, rotate, and zoom your photos right from the image block. If you spend a lot of time on images, this could save you hours!' ); ?></p>
+				<h3><?php _e('Block patterns' ); ?></h3>
+				<p><?php _e( 'New block patterns make it simple and fun to create complex, beautiful layouts, using combinations of text and media that you can mix and match to fit your story.' ); ?></p>
+				<p><?php _e( 'You will also find block patterns in a wide variety of plugins and themes, with more added all the time. Pick any of them from a single dropdown — just click and go!' ); ?></p>
```
[the remark this answers is not in the document]
Comment (marybaum) [reply src/wp-admin/about.php:121]:
```suggestion
				<p><?php _e( 'You will also find block patterns in a wide variety of plugins and themes, with more added all the time. Pick any of them from a single place — just click and go!' ); ?></p>
```

File: src/wp-admin/about.php [hunk 4]
```diff
@@ -40,166 +38,202 @@
...
 
-		<div class="about__section ">
-			<div class="column has-subtle-background-color">
-				<h2 class="is-section-header"><?php _e( 'Just for developers' ); ?></h2>
+		<div class="about__section has-2-columns">
+			<div class="column">
+				<h2><?php _e('Accessibility'); ?></h2>
+				<p><?php _e( 'Every release adds improvements to the accessible publishing experience, and that remains true for WordPress 5.5. ' ); ?></p>
+				<p><?php _e( 'Now you can copy links in media screens and modal dialogs with a button, instead of trying to highlight a line of text.' ); ?></p>
+				<p><?php _e( 'You can also move meta boxes with the keyboard, and edit images in WordPress with your assistive device, as it can read you the instructions in the image editor.' ); ?></p>
```
[the remark this answers is not in the document]
Comment (marybaum) [reply src/wp-admin/about.php:139]: We'll keep watch ...

File: src/wp-admin/about.php (hunk 1 above)
Comment (marybaum) [reply src/wp-admin/about.php:124]: Could go either way ...

**Pull request closed without merging**